packages/node_modules/nav-frontend-skjema/src/checkboks-panel-gruppe.tsx

```diff
@@ -24,21 +24,27 @@ export interface CheckboksPanelGruppeProps {
      * Objekt som beskriver skjema-feil, se 'skjemaelement-feilmelding.tsx'
      */
     feil?: SkjemaelementFeil;
+    /**
+     * Styling klasse for indre div for flere konfigurasjonsmuligheter.
+     */
+    divClassName?: string;
```

Comment:
Hm, ser kanskje for meg at vi heller legger til en `grid?: boolean;` prop litt senere hvor flex-styling osv. kommer ferdig ut-av-boksen. Da tror jeg det kan være uheldig å allerede ha introdusert denne propen som vi i så fall må deprecate. Hva om den indre div'en bare får et fast klassenavn, ala `inputPanelGruppe__inner`?

Reply:
Det vil fungere med fast klassenavn, men da vil det ikke være mulig å ha flere stylinger på skjemaene. 

Det vil uansett hjelpe, så om det er løsningen er det OK.

Reply:
Hm, vi burde kanskje legge til en vanlig `className` på interfacet her. Slik at man alltids kan legge til custom klasser på den ytre `SkjemaGruppe` containeren.

Reply:
Enig, men vi trenger allikevel en prop for div klassen i tillegg.

Reply:
Hvordan det? Tror det skal holde med en custom klasse på ytre container. Men vi burde sikkert hente ut `{ ...other }` fra `this.props` samtidig, og spre disse på den ytre containeren (`<SkjemaGruppe {...other}>`) slik at det evt. blir mulig å legge på `ID`-attributter osv.